test_report.py

```diff
@@ -377,11 +377,14 @@ def report_single_test(suite, test, tests, failure_msg=None):
                 suite.log.testfail(f"{test.name} COMPILE FAILED")
             elif test.crashed:
                 sf.write("CRASHED\n")
-                for btf in test.backtrace:
-                    suite.log.warn(f"+++ Next backtrace: {btf} +++")
-                    suite.log.warn(open(btf, 'r').read())
-                    suite.log.warn(f"+++ End of backtrace: {btf} +++\n")
-                suite.log.testfail(f"{test.name} CRASHED (backtraces produced)")
+                if len(test.backtrace) > 0:
+                    for btf in test.backtrace:
+                        suite.log.warn(f"+++ Next backtrace: {btf} +++")
+                        suite.log.warn(open(btf, 'r').read())
+                        suite.log.warn(f"+++ End of backtrace: {btf} +++\n")
+                    suite.log.testfail(f"{test.name} CRASHED (backtraces produced)")
+                else:
+                    suite.log.testfail(f"{test.name} CRASHED (script failed)")
```

Comment:
@zingale can we in an easy way print the content of `{test_name}.err.out` here? :)

Reply:
we should be able to do that.  Make an issue describing what you want.  We would either need to re-read the err file or store it as a string in the Test object for output.

Reply:
Thanks, got it. Opened in #106